### PR TITLE
Feature/arch 640 seed container

### PR DESCRIPTION
**Pull request by pascal89**
Created/ updated test files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,15 +25,14 @@ services:
...
     container_name: seed-container-name
-    # this host works for Windows by default.. that's why I chose this
     hostname: host.docker.internal
     environment:
       SQL_DATABASE: "database-to-seed"
 
     # windows users: comment out extra_hosts section
     # linux/mac/wsl users: make sure the extra_hosts section ISN'T commented out
     extra_hosts:
-      host.docker.internal: 172.17.0.1
```
Comment (HBTeun) [inline README.md:36]: Not sure this is correct looking at the comments

**Review by HBTeun — APPROVED** [review]
If you check the ip change approved

File: scripts/mssql-seed.sh [hunk 2]
```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+#set -euo pipefail
+PATH="/opt/mssql-tools/bin:$PATH"
 
 # Environmentals
 # Optional:
```
Comment (Mastermindzh) [inline scripts/mssql-seed.sh:6]: In [this](https://github.com/Inforitnl/mssql-seed-container/pull/1/files#diff-83fcb7286ae1a7a5f886c1e047bd6a6406b9bdbee797c40948bb7c92af33babcL27) for loop you might want to check for files starting with a number, assuming you want all files to be sequentially numbered.

**Review by Mastermindzh — APPROVED** [review]

File: scripts/mssql-seed.sh (hunk 2 above)
Comment (pascal89) [reply scripts/mssql-seed.sh:6]: I already mentioned this in the README that's why I didn't want to make it a requirement.

File: README.md (hunk 1 above)
Comment (pascal89) [reply README.md:36]: Better now?

**Pull request merged**